internal/gitstream/owners.go

```diff
@@ -13,15 +11,26 @@ type Owners struct {
 	Component string   `yaml:"component"`
 }
 
-func (o *Owners) getAssignee(ctx context.Context, gc *github.Client, userLogin string) (string, error) {
+func (o *Owners) getAssignee(userLogin string) (string, error) {
 
 	for _, approver := range o.Approvers {
 		if approver == userLogin {
 			return userLogin, nil
 		}
 	}
 
-	// User isn't in m/s OWNER file, select a random owner
+	return "", fmt.Errorf("%s is not in the %s file", userLogin, ownersFile)
+}
+
+func (o *Owners) getRandomAssignee() (string, error) {
```

Comment:
```suggestion
func (o *Owners) getRandomApprover() (string, error) {
```

Reply:
I don't think we should "tied" it to the implementation. We can easily decide that we also consider `Reviewers` and not just `Approvers`.

I perfer to keep it this way if you don't mind.

Also, I would like to keep consistent with the `getAssignee` method.

Reply:
> We can easily decide that we also consider Reviewers and not just Approvers.

I agree with that, however I don't think it should be this type's responsibility. `Owners` should not decide if an reviewer is a suitable assignee or not - another function should do that.

Reply:
Changed the method name to `getRandom()`. Caller is now using `owners.getRandom()`.